src/components/SharedComponents/ActivityCount.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import INatIcon from "components/INatIcon";
+import Body3 from "components/SharedComponents/Typography/Body3";
+import { View } from "components/styledComponents";
+import type { Node } from "react";
+import React from "react";
+import { t } from "i18next";
```

Comment:
I haven't run any examples of this, but I know you mentioned it before. We'll need to do a full audit of the app in that case, since I've been consistently using t from i18next.